app/index.html

```diff
@@ -57,7 +55,7 @@ <h1 class="margin jumbo">PALLADIO-CAD</h1>
       <div class="twothird">
         <h1>Platform for Architects</h1>
         <h5 class="padding-32">Complete tasks, archive files, and make payments.</h5>
-        <a target="_blank" href="https://ropsten.etherscan.io/token/0xbdbc336a80582478765ba2bbf36a0d03a396ae1a#balances">Token address</a><br><br>
```

Comment:
This looks like an error waiting to happen.  It would make more sense to set this target from MaxScript once your ABI is loaded so you don't need to update this value on recompile of contracts